libs/api/src/lib/graphql/comment/comment.ts

```diff
@@ -345,12 +353,38 @@ export const GraphQLPublicComment: GraphQLObjectType<PublicComment, Context> =
         })
       },
       calculatedRatings: {
-        type: new GraphQLList(GraphQLCalculatedRating)
+        type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(GraphQLCalculatedRating))),
+        resolve: createProxyingResolver(
+          async (
+            {calculatedRatings, id},
+            _,
+            {prisma: {commentRating}, loaders: {commentRatingSystemAnswers}}
+          ) => {
+            if (calculatedRatings) {
```

Comment:
already calculated for top level comments. Now also returns it for non top level comments. You were able to rate them before but it was never returned.